dev-docs/bidders/mobfoxpb.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: bidder
+title: Mobfox_PB
+description: Prebid Mobfox Bidder Adapter
+biddercode: mobfoxpb
+usp_supported: true
+schain_supported: true
+media_types: banner, video, native
```

Comment:
I see only `banner` and `video` media types as supported in Prebid Server. Perhaps this was a copy-paste leftover?

Reply:
@mansinahar Prebid.js adapter supports Native, also we have implementation for prebid-server, but it currently not properly tested